rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -364,6 +374,7 @@ def sample(
             SampleBatch.TERMINATEDS: np.array(is_terminated),
             SampleBatch.TRUNCATEDS: np.array(is_truncated),
             "weights": np.array(weights),
+            "n_steps": np.array(n_steps),
```

Comment:
Can we add a comment as to what effect this extra information has on the batch/loss/prio-weights update procedure afterwards?
My current guess would be: If we sample a 3-step tuple from the buffer and the loss comes back with a TD-error for that tuple of 2.3, then we update all covered 3 timesteps with that new weight? Or only the first timestep? I think this is an important detail to describe also in the docstring of this buffer itself.

Reply:
We can leave a comment here. The effect on the priority weight is ambiguous in my opinion as to what do we compare? The lower or the upper bound in the `n_step` interval?

Reply:
We update only the last one - as it is the one that was sampled. Following the paper in which experienced replay was introduced `j` is sampled and then the prior values are collected. This is implemented here.